appveyor.yml

```diff
@@ -19,14 +19,17 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - echo %PATH%
   - ruby --version
-  - gem update --system
+  - gem update --system || gem update --system || gem update --system
+  - gem install bundler --quiet --no-ri --no-rdoc || gem install bundler --quiet --no-ri --no-rdoc || gem install bundler --quiet --no-ri --no-rdoc
+  - update_rubygems
   - gem --version
-  - gem install bundler --no-ri --no-rdoc
-  - SET BUNDLE_GEMFILE=Gemfile.windows
   - bundler --version
+  - SET BUNDLE_IGNORE_CONFIG=true
+  - SET BUNDLE_FROZEN=1
+  - SET BUNDLE_WITHOUT=development:guard:maintenance:tools:integration:changelog:docgen:travis:style:omnibus_package:aix:bsd:linux:mac_os_x:solaris
 
 build_script:
-  - bundle install --without no_windows omnibus_package
+  - bundle install || bundle install || bundle install
```

Comment:
I would actually not do this anymore.  An old bundler was having issues and we did this as a desperate workaround.  gem update and gem install bundler done above should negate the need to do this.  In fact, I think chef should have it's silly repetitions removed as well.

I'm ok if you want to punt this to a different ticket - just leave us a note on your card in FLOW.